src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -389,6 +398,22 @@ public static String colorize(String string)
         return ChatColor.translateAlternateColorCodes('&', string);
     }
 
+    public static String StrictColorize(String string)
+    {
+       String string2 = ChatColor.translateAlternateColorCodes('&', string);
+        final Matcher matcher = REGEX.matcher(string2);
+        if (matcher.find())
+        {
+            final String filteredcolorize = matcher.replaceAll("&");
```

Comment:
Does this method work? As I recall, there should be 2 string in `replaceAll` method.

Reply:
yes, i tested it, if you do smite player &kk it will skip and replaces the chatcolor.magic with &

Reply:
basically if there is a blacklisted color, it gets replaced with the & and the color that aint blacklisted pass, if there is a blacklist it makes uses of filteredcolorize , or else returns the normal colorize.

Reply:
Understandable.